integration/tests/post_accountowners_test.go

```diff
@@ -0,0 +1,173 @@
+// +build integration
+
+package tests
+
+import (
+	"context"
+	openapi "github.com/asecurityteam/asset-inventory-api/client"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+)
+
+func LookupNoAccountOwnerInResponse(t *testing.T, needle string, when time.Time) {
+	t.Errorf("response does not contain account owner for resource %s at %s", needle, when)
+}
+
+func CheckAccountOwnersPresent(t *testing.T, accountOwnersExpected openapi.AccountOwner, changes openapi.CloudAssetChanges) {
+
+	ctx := context.Background()
+	ts := SampleAssetChanges().ChangeTime.Add(time.Minute) // best way I could think of to get time to query accounts by
+	spl := strings.Split(changes.Arn, "/")
```

Comment:
Can you please add TODO or FIXME comment here to change logic once we support ELB/ALB?